packages/flutter/test/material/elevated_button_theme_test.dart

```diff
@@ -228,7 +228,7 @@ void main() {
     });
   });
 
-  testWidgets('Material 3: Theme shadowColor', (WidgetTester tester) async {
+  testWidgets('Material 3 - ElevatedButton repsects Theme shadowColor', (WidgetTester tester) async {
```

Comment:
tiny nit: If we want to update the test message to follow the convention that we followed for other M2/M3 tests, I think it's 'Material3 - ...' / 'Material2 - ...':)